crates/router/src/connector/paypal.rs

```diff
@@ -1055,7 +1067,15 @@ impl api::IncomingWebhook for Paypal {
             .body
             .parse_struct("PaypalWebooksEventType")
             .change_context(errors::ConnectorError::WebhookEventTypeNotFound)?;
-        Ok(api::IncomingWebhookEvent::from(payload.event_type))
+        let outcome: paypal::DisputeOutcome =
+            request
```

Comment:
this will fail for payments & refunds as DisputeOutcome will be present in dispute web hooks